src/r-bridge/lang-4.x/ast/model/type.ts

```diff
@@ -138,6 +138,10 @@ export const enum RawRType {
 	 * https://github.com/REditorSupport/languageserver/pull/328
 	 */
 	ExprOfAssignOrHelp = 'expr_or_assign_or_help',
+	/**
+	 * Pre-4.0 version of expr_or_assign_or_help, see {@link RawRType.ExprOfAssignOrHelp} documentation
```

Comment:
this should link to the R documentation/update release notes (i weakly recall a discussion for the R language server regarding that)